src/main/java/org/assertj/core/api/AbstractLocalDateTimeAssert.java

```diff
@@ -68,9 +69,7 @@ protected AbstractLocalDateTimeAssert(LocalDateTime actual, Class<?> selfType) {
   public SELF isBefore(LocalDateTime other) {
```

Comment:
reminder: describe the new behavior in the javadoc

Reply:
Mentioning and explaining the breaking change should be done for all changed assertions, I know that's a bit of work but I think it is necessary.